res/css/views/right_panel/_PinnedMessagesCard.scss

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 Travis Ralston
+Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
goodbye, self. 

(I'm okay with this happening - I don't want any of the old CSS associated with my name)

Reply:
I mean, there is no old CSS here. All of it is brand new, I probably renamed the file to save me having to copy-paste the license :)